rae_hw/src/peripherals/mic.cpp

```diff
@@ -108,6 +108,26 @@ void MicNode::timer_callback() {
     }
 }
 
+void MicNode::applyLowPassFilter(std::vector<int32_t>& buffer) {
+    static float prevSampleLeft = 0.0f;
```

Comment:
I'm not sure if static/consts is needed here, I would move them to member variables

Reply:
But I guess this is a cosmetic change